plugins/Kaleidoscope-LEDEffects/src/kaleidoscope/plugin/Jukebox.cpp

```diff
@@ -20,10 +20,12 @@
 #include "kaleidoscope/device/device.h"    // for CRGB
 #include "kaleidoscope/plugin/TriColor.h"  // for TriColor
 
-kaleidoscope::plugin::TriColor JukeboxEffect(CRGB(0xc8, 0xe8, 0xee),  /* TM */
+kaleidoscope::plugin::TriColor JukeboxEffect(F("Jukebox"),
```

Comment:
To allow for setting a custom name for the led mode I would change this to a constructor argument, but I'm not sure how to update the header files to support that. The same holds tru for JukeboxAlternateEffect and MiamiEffect.

Reply:
I'm ok with not tackling that as part of this PR.